File: 2o periodo/lp1/p1/ex16.py
n= int(input('insira a quantidade de n na sequencia: '))
a= int(input('insira o n multiplo: '))
b= int(input('insira o n multiplo: '))
count=i=0
while count<n:
    if i%a==0 or i%b==0:
        print(i)
        count+=1
    i+=1
    

File: 2o periodo/lp1/p1/ex16.c
#include <stdio.h>
int main(){
    int n, a, b, cont;
    printf("insira a quantidade de numeros na sequencia: ");
    scanf("%d %d %d", &n, &a, &b);

    cont= 0;

    for (int i=0; cont<n; i++){
        if (i%a==0 || i%b==0){
            printf("%d ", i);
            cont++;
        }
    }

    return 0;
}
